freeipa/src/main/java/com/sequenceiq/freeipa/service/freeipa/user/UserServiceConstants.java

```diff
@@ -0,0 +1,11 @@
+package com.sequenceiq.freeipa.service.freeipa.user;
+
+public final class UserServiceConstants {
+
+    public static final String USERSYNC_INTERNAL_GROUP = "usersync-internal";
```

Comment:
maybe "cdp-users"?

Reply:
cdp-users will be misleading. I want to make sure that its clear in terms of "what its for"